docs/src/routes/docs/customElements.svx

```diff
@@ -0,0 +1,74 @@
+---
+name: Custom Elements
```

Comment:
why the camel case in the name? we usually name files with dashes

Reply:
yeah, I didn't look beyond the routes for reference)